src/libparsec/include/od_odt.h

```diff
@@ -1,480 +1,472 @@
-
-	OD2_Vertex3		BoundingCenter;		// center of bounding sphere
-	float			BoundingSphere; 	// radius of bounding sphere
-	OD2_CullBox		BoundingBox;		// axial bounding box
-};
-
-
-// child (non-root) node in ODT2 object graph ---------------------------------
-//
-struct OD2_Child {
-
-	dword			childflags;
-	dword			childflags2;
-
-	OD2_Node*		NodeList;			// first node in attached list
-	OD2_Child*		Children[ 2 ];		// child nodes in object graph
-
-	ODT_Xmatrx		NodeTrafo;	   		// object-space transformation (relative)
-};
-
-
-// ODT2 object's root node (includes file header) -----------------------------
-//
-struct OD2_Root {
-
-	char			odt2[ 6 ];			// signature ("ODT2")
-	byte			major;				// major revision
-	byte			minor;				// minor revision
-
-	dword			rootflags;
-	dword			rootflags2;
-
-	OD2_Node*		NodeList;			// first node in attached list
-	OD2_Child*		Children[ 2 ];		// child nodes in object graph
-
-	dword			ObjectType; 		// type this object belongs to
-	dword			ObjectClass;		// class this object belongs to
-	dword			InstanceSize;		// size of instance of this object class
-
-	dword			NumVerts;			// number of vertices w/ normals
-	dword			NumPolyVerts;		// number of vertices w/o normals
-	dword			NumNormals; 		// number of face normals
-	OD2_Vertex3*	VertexList;			// list of all vertices in object space
-
-	dword			NumPolys;			// number of polygons in this object
-	OD2_Poly*		PolyList;			// list of polygons in this object
-
-	dword			NumFaces;			// number of faces in this object
-	OD2_Face*		FaceList;			// list of all faces
-
-	dword			NumTextures;		// number of textures
-
-	OD2_Vertex3		BoundingCenter;		// center of bounding sphere
-	float			BoundingSphere; 	// radius of bounding sphere
-	OD2_CullBox		BoundingBox;		// axial bounding box
-};
-
-struct OD2_Root_Hdr {
-
-	char			odt2[ 6 ];			// signature ("ODT2")
-	byte			major;				// major revision
-	byte			minor;				// minor revision
-
-	dword			rootflags;
-	dword			rootflags2;
-
-	dword		NodeList;			// first node in attached list
-	dword		Children[ 2 ];		// child nodes in object graph
-
-	dword			ObjectType; 		// type this object belongs to
-	dword			ObjectClass;		// class this object belongs to
-	dword			InstanceSize;		// size of instance of this object class
-
-	dword			NumVerts;			// number of vertices w/ normals
-	dword			NumPolyVerts;		// number of vertices w/o normals
-	dword			NumNormals; 		// number of face normals
-	dword			VertexList;			// list of all vertices in object space
-
-	dword			NumPolys;			// number of polygons in this object
-	dword 		PolyList;			// list of polygons in this object
-
-	dword			NumFaces;			// number of faces in this object
-	dword			FaceList;			// list of all faces
-
-	dword			NumTextures;		// number of textures
-
-	OD2_Vertex3		BoundingCenter;		// center of bounding sphere
-	float			BoundingSphere; 	// radius of bounding sphere
-	OD2_CullBox		BoundingBox;		// axial bounding box
-};
-
-#endif // _OD_ODT_H_
-
-
+/*
```

Comment:
Looks like this file and `obj_odt.cpp` had their line endings changed from CRLF to LF, which makes diffs hard to read

Reply:
Yes. But would be great to let git do the line endings conversion on checkout to have native line-endings where ever you check it out.
There are always some tools that use <backslash><newline> as a line continuation and would not detect it if it finds <backslash><CR><LF> on a system that have <backslash><LF> as its newline definition.

Reply:
There's definitely plenty of options to choose from for standardizing line endings (and other things), but they should probably be discussed separately and in other changes rather than mixed in with this one.

Reply:
I agree with @slime73. Can you update the line endings to match the repo so we can see the changes?  This might have to be something we are more cognisant of or, in a separate thread, agree to change wholesale.

Reply:
Done.